sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,135 @@
+ - Enhance PI tracing with printing output arguments [19f5ad67f30a]
+ - Introduce `pi_map_flags` in lieu of `cl_map_flags` [f0e7606a6198]
+ - Implement robust error nadling in LevelZero plugin [65c719ddfc23]
+ - Add online compilation API interface [70ac47d23264]
+ - Add new device descriptors as SYCL extensions [51ac08c35294]
+ - Implement [`SYCL_INTEL_mem_channel_property`](doc/extensions/MemChannel/SYCL_INTEL_mem_channel_property.asciidoc)
+   extension [2f1f3167b7c6]
+ - Remove redundant dependencies for in-order queues [632722165db2]
+ - Add `marray` class as defined by SYCL 2020 provisional [5eebd1e4bfce]
+### Documentation
+ - Add information on AOT to GetStartedGuide [71942fbb3655]
+ - Add notice on alignemnt checks in ABI policy [4326b9563575]
+ - Updated design of specialization contants on using of POD types
+   [81963d1ec055]
+ - Document linked allocation commands [929a764a5ec4]
+ - Improve ESIMD documentation rendering [079597d28f1f]
+ - Improved device library documentation [f24e2a9ce464]
+ - Add online compilation specification [e05a19c8d303]
+
+## Bug fixes
+### SYCL Compiler
+ - Do not customize optimizations for non-SPIR targets [cb069fed6712]
+ - Fix address space assertion with templates [8905a8cec9a9]
+ - Link libm-fp64 device library by default [ac93d6fe3d9d]
+ - Add support for specialization constants' typenames declared in namespaces
+   [f64f835b4313]
+ - Fix loosing OpenMP device binary when program uses both OpenMP and SYCL
+   offloading models [eb89f5eaab37]
+### SYCL Library
+ - Add missing interoperability API to construct SYCL classes with Level-Zero
+   handles [10b4e8a6fc19]
+ - Fix several builtins implementation for host device
+   [8b82c671ab12, 786708914fd4]
+ - Fix possible hang upon application finish if streams were used [bd5893ae01b1]
+ - Fix failure when employing interoperability host task on queue constructed
+   with reused context [9cff6c9b6127]
+ - Fix "instantiation after specialization" warnings
+   [56b9a1dfb92f, eadce94f8ad0]
+ - Support copying of stream by value within a kernel without loss of output
+   information [8d37cbacc9b8]
+ - Fix handling of big and/or non-uniform work-groups in reduction kernels. The
+   solution may change when reduction kernels precompilation/query approach is
+   implemented [78e2599bc499]
+ - Fix memory leak in event pool in Level Zero plugin [68fc7808a50e]
+ - Fixed issue with finalizing context of Level Zero plugin [6cfa921856f5]
+ - Fix backend selection for `SYCL_DEVICE_FILTER=*` case [c54da157f5d5]
+ - Restore AccessorImplHost layout [a08eeb475679]
+### Documentation
+ - Updated source checkout instruction for Windows in GetStartedGuide
+   [9cde15210d70]
+
+## API/ABI breakages
+
+## Known issues
+  - GlobalWorkOffset is not supported by Level Zero backend [6f9e9a76]
+  - The code with function pointer is hanging on Level Zero [d384295e]
+  - If an application uses `std::*` math function in the kernel code the
```

Comment:
Hi, we have enabled fp64 math device library default link in https://github.com/intel/llvm/pull/2892
After that, -fsycl-device-lib=libm-fp64 is not required anymore, we can remove this known issue.
Thanks very much.

Reply:
Removed.